gensim/models/word2vec.py

```diff
@@ -606,7 +606,7 @@ def load_word2vec_format(cls, fname, fvocab=None, binary=False, norm_only=True):
                     result.syn0[line_no] = fromstring(fin.read(binary_len), dtype=REAL)
             else:
                 for line_no, line in enumerate(fin):
-                    parts = utils.to_unicode(line).split()
+                    parts = utils.to_unicode(line.rstrip()).split(" ")
```

Comment:
`rstrip` will have the same problems as `strip`, it's the same code. If it's the newline that creates problems, remove the newline only.